packages/@aws-cdk/pipelines/README.md

```diff
@@ -481,6 +481,17 @@ pipeline.addStage(prod, {
 });
 ```
 
+You can also approve changeSets between a stacks `prepare` and `deploy` phases. To achieve this, you can specify a `changeSetApproval` property on a `Stage`:
+
+```ts
+pipeline.addStage(prod, {
+  changeSetApproval: [{
+    step: new ManualApprovalStep('CheckChangeSet'),
+    stacks: [ProdStack1, ProdStack2],
```

Comment:
A simple solution around the 2nd, without having to make a decision now, is to type it as:

```ts
readonly stack: Stack
```

So not make it an array. Then it can only mean one thing, and potentially we could still add the array API later.